client.go

```diff
@@ -217,6 +219,19 @@ func (m *PulsarAPI) client(c sobek.ConstructorCall) *sobek.Object {
 		common.Throw(rt, err)
 	}
 
+	if subscriptionMode := c.Argument(14); isNilOrUndefined(subscriptionMode) {
+		clientConf.subscriptionMode = pulsar.NonDurable
+	} else {
+		if subscriptionMode.String() != "Durable" && subscriptionMode.String() != "NonDurable" {
+			common.Throw(rt, fmt.Errorf("invalid subscription mode %s, must be 'Durable' or 'NonDurable'", subscriptionMode.String()))
+		}
+		if subscriptionMode.String() == "Durable" {
+			clientConf.subscriptionMode = pulsar.Durable
+		} else {
+			clientConf.subscriptionMode = pulsar.NonDurable
+		}
+	}
+
```

Comment:
May be we can move this to the inputs of the subscribe method, similar to the susbscriptionType?